docs/fsharp/style-guide/formatting.md

```diff
@@ -1216,6 +1216,102 @@ In single-line expressions the delimiter symbols should be placed on the same li
 @>
 ```
 
+### Formatting chained expressions
+
+When chained expressions (function applications intertwined with `.`) are long, put each application invocation on the next line.
+Indent the subsequent links in the chain by one level after the leading link.
+
+```fsharp
+// ✔️ OK
+Host
+    .CreateDefaultBuilder(args)
+    .ConfigureWebHostDefaults(fun webBuilder -> webBuilder.UseStartup<Startup>())
+
+// ✔️ OK
+Cli
+    .Wrap("git")
+    .WithArguments(arguments)
+    .WithWorkingDirectory(__SOURCE_DIRECTORY__)
+    .ExecuteBufferedAsync()
+    .Task
+```
+
+The leading link can be composed out of multiple links if they are simple identifiers.
+For example, the addition of a fully qualified namespace.
+
+```fsharp
+// ✔️ OK
+Microsoft.Extensions.Hosting.Host
+    .CreateDefaultBuilder(args)
+    .ConfigureWebHostDefaults(fun webBuilder -> webBuilder.UseStartup<Startup>())
```

Comment:
Why can't it contain method calls too?
The following code should be allowed by de style guide:

```fsharp
Microsoft.Extensions.GetHosting().Host
     .CreateDefaultBuilder(args)
     .ConfigureWebHostDefaults(fun webBuilder -> webBuilder.UseStartup<Startup>())
```

Reply:
@auduchinok could you please join the conversation over at https://github.com/fsharp/fslang-design/issues/688 when it is taking place? This is frustrating that you question the decision of something that was labelled `style-guide-decided`. In the future please, subscribe to that repository and engage when the solution is being discussed.

To answer your question, the reasoning was that having the method calls always at the end would be more readable and make it clear when invocations are happening.

Reply:
>  In the future please, subscribe to that repository and engage when the solution is being discussed.

I am subscribed to it.

> @auduchinok could you please join the conversation over at https://github.com/fsharp/fslang-design/issues/688 when it is taking place?

Yes I could. I've seen it, and this example was not discussed there. I'm commenting a specific place in the PR in the appropriate place.